pizza-recipes.md

```diff
@@ -174,3 +174,63 @@ Embrace this blend of flavors as you forgo store-brought sauce for this nuturiou
 6. Cook for 30 mins, adjust water if thick.
 
 7. Let it cool. Your pizza sauce is ready to use.
+
+
+# Chicken Tikka Masala Pizza ( India )
```

Comment:
[suggestion] Since you're describing a recipe. You don't need to mention that it's from India. 
```suggestion
# Chicken Tikka Masala Pizza 
```

Reply:
@CBID2 I have done changes for my end and please do not forget to add the hactoberfest tag😀